<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorTextEdit</class>
 <widget class="QDialog" name="SensorTextEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Text for Sensor States</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="nameText">
     <property name="text">
      <string>xxxxx</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textInact">
     <property name="text">
      <string>Inactive = Inactive</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="inactTextField">
     <property name="text">
      <string>Inactive</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textAct">
     <property name="text">
      <string>Active = Active</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="actTextField">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textIncon">
     <property name="text">
      <string>Inconsistent - &lt;inconsistent&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="inconTextField">
     <property name="text">
      <string>&lt;inconsistent&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textUnknown">
     <property name="text">
      <string>Unknown= &lt;unknown&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="unknownTextField">
     <property name="text">
      <string>&lt;unknown&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
